chronosphere/resource_resource_pools_config.go

```diff
@@ -301,6 +360,30 @@ func buildFixedValues(fixedValues []intschema.ResourcePoolAllocationSchemaFixedV
 	})
 }
 
+func buildThresholds(thresholds []intschema.ResourcePoolAllocationThresholdsSchema) []*apimodels.AllocationThresholds {
```

Comment:
for these types of conversion function we usually use a convention of `ToModel` and `FromModel` -- you should be able to find lots of examples in the code. I think that keeping that consistency would help

Reply:
Renamed everything accordingly